recipe/bld.bat

```diff
@@ -0,0 +1,62 @@
+
+mkdir third_party\boost\preprocessor\include
```

Comment:
DALI doesn't support Windows at the moment as there are places that are Linux-specific and won't work on Windows by definition (path separators, specific OS syscalls).